terraform/ai-answers.alpha.canada.ca.tf

```diff
@@ -0,0 +1,19 @@
+resource "aws_route53_record" "ai-answers-alpha-canada-ca-CNAME" {
+  zone_id = aws_route53_zone.alpha-canada-ca-public.zone_id
+  name    = "ai-answers.alpha.canada.ca"
+  type    = "CNAME"
+  records = [
+    "react-answers.vercel.app"
```

Comment:
I'm not sure this should be hosted in Vercel, I'd be okay if it was hosted in our secure cloud environments instead

Reply:
I believe that if this product becomes live, it will need an ATO (could be a short term one), and a migration to a different cloud will likely be required.

Reply:
I am going to take a hard stance and say I will not approve this until it migrates to a Secure cloud enviroment

Reply:
It's not just Vercel I have an issue with it's the underlying API's that are in use, it needs to be migrated to Azure's Open AI API's.

Reply:
+1